mu-plugins/blocks/local-navigation-bar/postcss/style.pcss

```diff
@@ -110,8 +110,17 @@
 		position: relative !important;
 		top: 0 !important;
 
-		/* Matches the padding of the global header button. */
-		padding-right: calc(16px + var(--wp--custom--alignment--scroll-bar-width)) !important;
+		/* Needed for aligning the dropdown caret correctly. */
+		padding-right: 0;
+
+		/* Add right padding if the dropdown caret is not being displayed. */
+		&:not(.wporg-show-collapsed-nav) {
+			& .wp-block-navigation {
+				&:not(.wporg-is-mobile-nav) {
+					padding-right: calc(16px + var(--wp--custom--alignment--scroll-bar-width));
+				}
+			}
+		}
```

Comment:
This still has the double-padding issue:

<img width="538" alt="Screenshot 2024-05-02 at 10 24 16 AM" src="https://github.com/WordPress/wporg-mu-plugins/assets/541093/5542ef6f-dd7c-4f6b-9f8a-52ecec33cfe6">

If you add the `nav` element, it should apply to just the top-level class. Also, previously this matched the content alignment (`edge-space`) not the icon, and I don't know if the change to match the icon when it's text was intentional or not. I don't have a strong opinion either way, it's only 4px :)

```suggestion
		&nav:not(.wporg-is-mobile-nav):not(.wporg-is-collapsed-nav) {
			padding-right: var(--wp--preset--spacing--edge-space); // or the calculated one
		}
```

Reply:
```
&nav:not(.wporg-is-mobile-nav):not(.wporg-is-collapsed-nav) {
        padding-right: var(--wp--preset--spacing--edge-space); // or the calculated one
}
```
This did not work for me, but `& nav:not....` did. Just updated the PR with this change.